### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by kimstik** [free-pdk/easy-pdk-showcase-projects#2]
I would like to suggest several projects that it would be fun to see in pdk:

https://www.quinapalus.com/picsi.html
https://www.dos4ever.com/uscope/uscope_e.html

BTW Thanks for noiseplug ;)

**Comment by freepdk** [comment]
I had a brief look at the projects.

1) https://www.quinapalus.com/picsi.html
This project uses a much to "powerful" PIC16F648A (FLASH:4096 (2048 words) EEPROM:256  Byte RAM:256 Byte)

>Almost all the resources of the PIC microcontroller are used: 99.9% of the program memory (2046 out of 2048 locations), all but two of the data EEPROM locations, and almost all of the data RAM. 

So even the most powerful PADAUK might not be able to handle this.

2) https://www.dos4ever.com/uscope/uscope_e.html
This might be possible, but looks a bit "boring" and I think not many people will have a classic "TV" with analog input anymore....


However, a project like this should be possible: 

https://github.com/benheck/attiny10_game
https://www.youtube.com/watch?v=PZsWqOuJFKI
https://www.youtube.com/watch?v=aq30EdgSpOY

**Comment by kimstik** [comment]
You are right.  Composite video actually a bit less popular than display-port ;)
PFS173 seems perfect candidate for "powerful" applications
